### ISSUE TITLE
no app_version information in fronted: harbor_version is empty on portal

### ISSUE THREAD

**Issue by MatthiasHertel** [bitnami/bitnami-docker-harbor-portal#19]
**Description**

**Steps to reproduce the issue:**

install harbor via helmchart

**Describe the results you received:**

![image](https://user-images.githubusercontent.com/8426566/127103313-50b7d9ab-7a15-4be4-9b95-35e4b48c5268.png)

![image](https://user-images.githubusercontent.com/8426566/127103278-0d0e41a2-ce99-43a4-bcde-a70b544e335d.png)

**Describe the results you expected:**

a version string is shown in the frontend

**Additional information you deem important (e.g. issue happens only occasionally):**

i also created a issue in core repo

https://github.com/goharbor/harbor/issues/15128

**Version**

```
λ helm search repo bitnami/harbor --versions
NAME            CHART VERSION   APP VERSION     DESCRIPTION
bitnami/harbor  10.2.3          2.3.1           Harbor is an an open source trusted cloud nativ...

```

- Output of `docker info`:

```
(paste your output here)
```

- Output of `docker-compose version` (if applicable):

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

**Comment by marcosbc** [comment]
Hi @MatthiasHertel, this is apparently due to how we compile `harbor_core` as no version info is added at that stage:

```console
$ harbor_core
...
2021-08-12T14:08:47Z [INFO] [/core/main.go:237]: Version: , Git commit:
...
```

If you need to obtain the Harbor version, you can always execute the following command:

```console
$ cat /opt/bitnami/harbor-core/UIVERSION
2.3.1
```

**Comment by MatthiasHertel** [comment]
hi @marcosbc 

thx for your answer.

hmm is it possible to add the version info in the compile stage ?
i guess it would be comfortable from UX point of view if the user can see which harbor version he use.

kind regards
matthias

**Comment by FraPazGal** [comment]
Hello @MatthiasHertel,

This issue is now fixed in the latest chart's major, `12.0.0`. You can check it so by doing:

```
$ helm repo update
$ helm search repo bitnami/harbor
NAME          	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/harbor	12.0.0       	2.4.1      	Harbor is an open source trusted cloud-native r...
$ helm install harbor bitnami/harbor
```

<img width="246" alt="Screenshot 2022-03-09 at 13 43 32" src="https://user-images.githubusercontent.com/26460210/157444085-5bc7ca64-f912-4ff2-9e40-449ed6ee4add.png">

Thanks for reporting this bug and please do not doubt to reach out to us if there is any further issue.

**Comment by MatthiasHertel** [comment]
Hi @FraPazGal 

first of all - thank u for fixing this bug. :+1: 
i will give the new version a shot and will let u know if it solved for me
